acl/tests/test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
Please use the standard header for test.sh as follows:

```
#!/bin/sh
#/ Usage: test.sh <pkg_ident>
#/
#/ Example: test.sh core/opa/0.11.0/20190531065119
#/```

Reply:
Amended.

Reply:
Amended